lib/main.coffee

```diff
@@ -29,7 +39,7 @@ module.exports =
 
     # Check version of coala
     version = helpers.exec(@executable, ['--version']).then (result) ->
-      if result <= COALA_MIN_VERSION
+      if cmpVersions(result, COALA_MIN_VERSION) < 0
```

Comment:
Can we use inbuilt functions by importing from a library? Better to not reinvent the wheel :)